test/vm-upload

```diff
@@ -53,7 +54,7 @@ def upload(store, source):
     ret = curl.wait()
     cat.wait()
     if ret != 0:
-        sys.stderr.write("vm-upload: unable to upload image\n")
+        sys.stderr.write("image-upload: unable to upload image: {0}\n".format(message))
```

Comment:
The command is really being called "vm-upload", so I'd revert the first bit. `message` is not guaranteed to be defined as `ai` might be empty; but then it would already crash in line 36 on `resolve`, so that part is fine after all.